itou/siae_evaluations/fixtures.py

```diff
@@ -0,0 +1,157 @@
+from django.test import override_settings
+from django.utils import timezone
+
+
+django.setup()
+
+import itou.siae_evaluations.enums as evaluation_enums
+import itou.users.enums as users_enums
+from itou.approvals.models import Approval
+from itou.eligibility.enums import AdministrativeCriteriaLevel
+from itou.eligibility.models import AdministrativeCriteria
+from itou.eligibility.models.iae import EligibilityDiagnosis
+from itou.institutions.models import Institution
+from itou.job_applications.models import JobApplication, JobApplicationWorkflow
+from itou.siae_evaluations.models import EvaluationCampaign
+from itou.siaes.models import Siae
+from itou.users.models import User
+
+
+NAMES = [
+    ("Gustave", "Loiseau", "187104712403340", "703059J"),
+    ("Aristide", "Filoselle", "187103415217885", "401486E"),
+    ("Oliveria", "Da Figueira", "187102017128676", "354471C"),
+    ("Fan", "Se-Yeng", "187102531808302", "145793B"),
+    ("Archibald", "Haddock", "187107317012993", "103819B"),
+    ("Bianca", "Castafiore", "287103404102922", "50704F"),
+    ("Mohammed", "Ben Kalish Ezab", "187105714612783", "476261C"),
+    ("Tryphon", "Tournesol", "187105012012584", "321089E"),
+    ("Bunji", "Kuraki", "187101300806442", "298096C"),
+    ("Roberto", "Rastapopoulos", "187107922316551", "424548G"),
+    ("Séraphin", "Lampion", "187104308500783", "491405D"),
+    ("Allan", "Thompson", "187101033733471", "391055C"),
+    ("Diego", "le Navarrais", "187101703338725", "718757J"),
+    ("Igor", "Wagner", "187107823832946", "621359J"),
+    ("Frank", "Wolff", "187106129833161", "644055J"),
+    ("Tristan", "Bior", "187106539602337", "355357E"),
+    ("Émile", "Vanneau", "187102117208427", "354895F"),
+    ("Jean-Loup", "de La Batellerie", "187109413107670", "354895D"),
+]
+
+
+def generate_approval_number():
+    return f"{Approval.ASP_ITOU_PREFIX}" + f"{random.randint(1000, 9999999)}".zfill(7)
+
+
+now = timezone.now()
+evaluated_period_start_at = now - relativedelta(months=3)
+evaluated_period_end_at = now - relativedelta(months=1)
+datetime_within_period_range = evaluated_period_end_at - relativedelta(weeks=2)
+
+created_job_applications_pks = []
+
+controlled_siaes = Siae.objects.filter(pk__in=[2653, 7, 184, 249, 250])
+employer = User.objects.get(email="test+cap@inclusion.beta.gouv.fr")
+total_administrative_criteria = AdministrativeCriteria.objects.count()
+level_to_criteria_pks = dict(
+    AdministrativeCriteria.objects.values("level").annotate(pks=ArrayAgg("pk")).values_list("level", "pks")
+)
+
+users = User.objects.filter(username__startswith="siae_evaluations_")
+with transaction.atomic():
+    users.delete()
+    EvaluationCampaign.objects.all().delete()
+
+siae_index = 0
+# We can't use factories here because FactoryBoy is not installed in Review app and Demo environments.
+for i in range(1, total_administrative_criteria):
+    siae_index += 1 if i % 4 == 0 else 0
+    controlled_siae = controlled_siaes[siae_index]
```

Comment:
```suggestion
    controlled_siae = controlled_siaes[i // 4]
```
semble faire grosso modo la même chose :grin:

Reply:
Ooooh ! :star_struck: